understanding/21/label-in-name.html

```diff
@@ -9,23 +9,61 @@
-			<p>This issue is even more important for speech input and text-to-speech users who also have cognitive challenges. It’s an extra
-			cognitive load for a speech input user to remember and say a speech command that is different from the visible label they see on a
-			control. It’s also an extra cognitive load for a text-to-speech user to absorb and understand speech output that does not
-			match the visible label.</p>
+			<p>The intent of this Success Criterion (SC) is to make text that is <em>visually</em> associated with a component also associated with the component <em>programmatically</em>. This helps ensure that people with disabilities can rely on visible labels as a means to interact with the components.</p>
+			<p>Most controls are accompanied by a visible text <strong>label</strong>. Those same controls have a programmatic <strong>name</strong>, also known as the <a href="https://www.w3.org/TR/accname-aam-1.1/">Accessible Name</a>. Users typically have a much better experience if the words in the visible label of a control match (or are contained within) the accessible name. When these match, speech-input users (i.e., users of speech recognition applications) can navigate by speaking the visible text labels of components, such as menus, links and buttons, that appear on the screen. Sighted users who use text-to-speech (e.g., screen readers) will also have a better experience if the text they hear matches the text they see on the screen.</p>
+			<p>Note that where a visible text label does not exist for a component, this Success Criterion does not apply to that component.</p>
+			<p>Where text labels exist and are properly linked to the user interface components through established authoring practices, the label and name will normally match. When they don't match, speech-input users who attempt to use the visible text label as a means of navigation or selection (e.g., "move to Password") will be unsuccessful.  The speech-based navigation fails because the visible label spoken by the users does not match (or is not part of) the accessible name that is enabled as a speech-input command. In addition, when the accessible name is different from the visible label, it may function as a hidden command that can be accidentally activated by speech-input users.</p>
+			<p>Mismatches between visible labels and programmatic names for controls are even more of an issue for speech-input and text-to-speech users who also have cognitive challenges. Mismatches create an extra cognitive load for speech-input users, who must remember to say a speech command that is different from the visible label they see on a control. It also creates extra cognitive load for a text-to-speech user to absorb and understand speech output that does not match the visible label.</p>
+			<h3>Confining the label to adjacent text</h3>
```

Comment:
This was the one from the call where we suggested "Identifying label text for components"

Reply:
Okay, I had modified but hadn't quite matched your words. Now updated.